src/JsCube.js

```diff
@@ -1,24 +1,27 @@
 var nj = require('numjs');
 
 const dimensionTypes = Object.freeze({
```

Comment:
@kostaricic Can you please extract this into a separate file, and name it something like `enum.js`? Then import it here with something like `const dt = require(`enum');`... This will expand, so we need a separate place for it.